apps/condo/pages/property/[id]/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { useIntl } from '@core/next/intl'
+import React from 'react'
+import { Row, Col, Typography, Tag, Space } from 'antd'
+import { useRouter } from 'next/router'
+import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { PageContent, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+import { useObject } from '@condo/domains/property/utils/clientSchema/Property'
+import { LinkWithIcon } from '@condo/domains/common/components/LinkWithIcon'
+import { ArrowLeftOutlined, EditFilled } from '@ant-design/icons'
+import { colors } from '@condo/domains/common/constants/style'
+import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
+import Head from 'next/head'
+import Link from 'next/link'
+import { Button } from '@condo/domains/common/components/Button'
+import { FocusContainer } from '@condo/domains/common/components/FocusContainer'
+import { PropertyPanels } from '@condo/domains/property/components/panels/index'
+interface IPropertyInfoPanelProps {
+    title: string
+    message: string
+    type?:  'success' | 'warning'
+}
+
+const PropertyInfoPanel: React.FC<IPropertyInfoPanelProps> = ({ title, message, type }) => {
+
+    return (
+        <FocusContainer style={{ margin: 'initial', width: '180px', height: '105px' }}>
+            <Space direction={'vertical'} size={8}>
+                <Typography.Text type={'secondary'}>{title}</Typography.Text>
+                <Typography.Text {...{ type }} style={{ fontSize: '20px', fontWeight: 'bold' }}>{message}</Typography.Text>
+            </Space>
+        </FocusContainer>             
+    )
+
+}
+
+interface IPageWithHeaderAction extends React.FC {
+    headerAction?: JSX.Element
+}
+
+const PropertyIdPage: IPageWithHeaderAction = () => {
+    const intl = useIntl()
+    const PageTitleMsg = intl.formatMessage({ id: 'pages.condo.property.id.PageTitle' })
+    const ServerErrorMsg = intl.formatMessage({ id: 'ServerError' })
+
+    const router = useRouter()
+    let { query: { id } } = router
+    if (typeof id !== 'string') {
```

Comment:
can u, please, add same check to `apps/condo/pages/ticket/[id]/index.tsx:149` and remove type cast?

Reply:
I don't know how to fix it without wrighting lot's of typescript code redeclaring all starting from NextRouter. If you have some suggestions - they are welcome
```js
const { query: { id } } = useRouter()
const { loading, obj: property, error } = useObject({ where: { id: id as string } })
```

Reply:
`if (typeof id !== 'string') {
    id = String(id)
  }` 
   Its ok